core/test/qa_Settings.cpp

```diff
@@ -141,24 +133,24 @@ const boost::ut::suite SettingsTests = [] {
         expect(eq(block1.settings().getStored().value().size(), 13UZ));
         expect(eq(block1.name, "SettingsChangeRecorder#1"s));
         expect(eq(block1.settings().getNAutoUpdateParameters(), 1UZ));
-        expect(eq(block1.settings().autoUpdateParameters().size(), 8UL));  // 2 base + 6 derived
-        expect(eq(block1.settings().autoForwardParameters().size(), 2UL)); // sample_rate, context
+        expect(eq(block1.settings().autoUpdateParameters().size(), 8UL));   // 2 base + 6 derived
+        expect(eq(block1.settings().autoForwardParameters().size(), 16UL)); // all gr::tag::kDefaultTags
 
         auto& sink = testGraph.emplaceBlock<Sink<float>>();
         expect(eq(sink.settings().defaultParameters().size(), 9UZ)); // 7 base + 2 derived
         expect(eq(sink.settings().getNStoredParameters(), 1UZ));
         expect(eq(sink.settings().getStored().value().size(), 9UZ));
         expect(eq(sink.settings().getNAutoUpdateParameters(), 1UZ));
-        expect(eq(sink.settings().autoUpdateParameters().size(), 5UL));  // 3 base + 2 derived
-        expect(eq(sink.settings().autoForwardParameters().size(), 1UL)); // sample_rate
+        expect(eq(sink.settings().autoUpdateParameters().size(), 5UL));   // 3 base + 2 derived
+        expect(eq(sink.settings().autoForwardParameters().size(), 16UL)); // all gr::tag::kDefaultTags
```

Comment:
Using `gr::tag::kDefaultTags.size()` would be more expressive, avoid the code comment, and simplify later refactoring.